dspace-api/src/main/resources/spring/spring-dspace-core-services.xml

```diff
@@ -31,8 +31,14 @@
         <property name="configurationService" ref="org.dspace.services.ConfigurationService"/>
     </bean-->
 
-    <bean id="org.dspace.identifier.VersionedHandleIdentifierProvider"
-          class="org.dspace.identifier.VersionedHandleIdentifierProvider"
+<!--    <bean id="org.dspace.identifier.VersionedHandleIdentifierProvider"-->
```

Comment:
please remove or leave a comment, why it's necessary to have it commented out

Reply:
Added the comments why I commented it out, check the updated code: https://github.com/dataquest-dev/DSpace/blob/feature/laf-4-item-new-version/dspace-api/src/main/resources/spring/spring-dspace-core-services.xml#L34